boto3/docs/collection.py

```diff
@@ -271,7 +270,7 @@ def document_collection_method(
                     name='count',
                     type_name='integer',
                     documentation=(
-                        'The number of items returned by each ' 'service call'
```

Comment:
Just out of sheer curiosity, do you know why/how this one happened?  Same with the one below.

I'm guessing it's just a formatting error, but curious if there may be some other reason.

Reply:
Older versions of black did this prior to us moving to ruff. It's collapsing multiple lines but didn't remove the quotes on each side. Ruff wasn't catching these on our previous version either.